Phoenix/PhoenixDocument+Extensions.swift

```diff
@@ -17,7 +17,9 @@ extension PhoenixDocument: FileDocument {
     public init(configuration: ReadConfiguration) throws {
         if configuration.file.isDirectory, let fileWrapper = configuration.file.fileWrappers {
             let phoenixDocumentFileWrappersDecoder = Container.phoenixDocumentFileWrappersDecoder()
-            self = try phoenixDocumentFileWrappersDecoder.phoenixDocument(from: fileWrapper)
+            let phoenixDocument = try phoenixDocumentFileWrappersDecoder.phoenixDocument(from: fileWrapper)
+            self = phoenixDocument
+            print(self)
```

Comment:
I'm guessing this change is not necessary?

Reply:
Reverted in 41fd94518c094dc26f933268684b7019677128fd